docs/DevGuide/CodeReviewGuide.md

```diff
@@ -53,6 +52,9 @@ Stylistic Items:
   `or`, `and`, and `not` instead of `||`, `&&`, and `!`.
 * Use C-style Doxygen comments (`/*! ... */`) when using multi-line math,
   otherwise C-style and C++ style comments are accepted.
+* Use the `align` environment instead of `equnarray`.
```

Comment:
Suggest adding this link, which has an example of the difference along with why not to use eqnarray https://texfaq.org/FAQ-eqnarray